### PR TITLE
Refactor some of the server code

### PR DESCRIPTION
**Pull request by Brystephor**
A lot of the code is copy and pasted with very small changes. This is refactoring some of that code into a more reusable format and consistent format. There is also a lot of 'magic strings' with no centralized location for making the strings easy to access. Adding constants where we can see what the types will be makes adding and extending changes much much easier. It'd be great to get this game to a state where we can add more roles from expansions.

### CODE REVIEW

File: coup-client/package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
   },
   "scripts": {
     "start": "PORT=3000 react-scripts start",
+    "start-pc": "set PORT=3000 && react-scripts start",
```
Comment (Brystephor) [inline coup-client/package.json:23]: This makes it easy for windows users to start the client.

File: server/game/utils.js [hunk 2]
```diff
@@ -1,19 +1,10 @@
+const constants = require("../utilities/constants");
+
 buildDeck = () => {
     let deck = []
-    for(let i = 0; i < 3; i++) {
```
Comment (Brystephor) [inline server/game/utils.js:3]: This is an example of copy and pasted code when a function would be much better.

**Comment by Cheneth** [comment]
I'll take a look at it soon, sorry just a little busy at the moment.

**Comment by Brystephor** [comment]
If you have a bug list, desired features list, or roadmap other than the
issues tab for the repo then it'd be great to share that. I like the games
web interface a lot. A few quality of life improvements on it would make it
fantastic.

**Comment by Cheneth** [comment]
As of right now, there's no bug list other than the issues tab, but I do have a mental list of some potential features:

- Make the events log scrollable
- Create a mechanism to allow users to rejoin rooms if they use the same name
- Maybe add support for the expansion
- Maybe add icons instead of just colored circles (although that the circles might actually be better)

**Pull request merged**